### PR TITLE
initial pass at an introduction to software design and its importance

### PR DESCRIPTION
**Pull request by harrygfox**
Initially from issue #94

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,6 +212,12 @@ we suggest you check out the following:
 + Interactive Workshop: https://github.com/jlord/git-it-electron
 + [@NataliaLKB](https://github.com/NataliaLKB)'s Tutorial: https://github.com/NataliaLKB/learn-git-basics
 
+### Design
+
+Building something for _someone else_ is hard. You don't necessarily know that people. Worrying about someone else's _"experience"_ with your app can be a full-time job. However, it is an essential process for everyone involved in software to partake in. The very best pieces of software are differentiated by the quality of their **User Experience design**.
```
Comment (newswim) [inline README.md]: I'd change "people" -> "person"

File: README.md [hunk 2]
```diff
@@ -212,6 +212,12 @@ we suggest you check out the following:
...
+
+ Experiences are, by nature, personal, contextual, ephemeral, and affected by things beyond our control. The job of designing one is essentially one of embodying the _real people_ that will be using your product and imagining what the experience would be like for them. You should try to take into account as much contextual information about your users state of mind, occupation, preference towards sans-serif fonts etc. These details will weave a picture of the real humans on the other end of your app and help you make decisions about site structure and navigation as other less obvious things like content hierarchy: _what should the user see first?_
```
Comment (newswim) [inline README.md]: "beyond our control" -> "beyond your control" -- "You" is used throughout the rest of the paragraph.

**Comment by newswim** [comment]
Few small typo things:

1st Paragraph:
Said "people", but I think you meant "person"

2nd paragraph:
"beyond our control" -> "beyond your control" -- "You" is used throughout the rest of the paragraph.

Great content!

File: README.md [hunk 3]
```diff
@@ -212,6 +212,12 @@ we suggest you check out the following:
 + Interactive Workshop: https://github.com/jlord/git-it-electron
 + [@NataliaLKB](https://github.com/NataliaLKB)'s Tutorial: https://github.com/NataliaLKB/learn-git-basics
 
+### Design
```
Comment (nelsonic) [inline README.md]: @harrygfox looks good. 😍 
I'd _suggest_ splitting out the sentences into new lines for ease of reading. see: http://rhodesmill.org/brandon/2012/one-sentence-per-line
@newswim thanks for reviewing. ❤️

File: README.md (hunk 2 above)
Comment (nelsonic) [inline README.md]: @harrygfox looks good. 😍 
I'd _suggest_ splitting out the sentences into new lines for ease of reading. see: http://rhodesmill.org/brandon/2012/one-sentence-per-line
@newswim thanks for reviewing. ❤️

File: README.md (hunk 1 above)
Comment (iteles) [inline README.md]: As @newswim mentioned, "that people" :arrow_right: "that person" or "those people"

**Comment by harrygfox** [comment]
@newswim @nelsonic @iteles 
I've addressed your suggestions 👍

**Comment by nelsonic** [comment]
Merging as both reviewers have approved. 👍

**Pull request merged**